### ISSUE TITLE
issue: Instructions ESPhome after Lovelace UI not clear

### ISSUE THREAD

**Issue by Black-Smokey** [Blackymas/NSPanel_HA_Blueprint#559]
Hello,

First of all, thank you for creating this wonderfull solution.

My problem:

I've experimented with both solutions: Lovelace UI and HA_Blueprint. HA_Blueprint has a better fit with my wishes so i would like to change my other NSPanel to HA_Blueprint. Unfortunately i don't get it done.

I read the instructions from https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-Common-Issues-TFT-Upload-When-nspanel-lovelace-ui-has-been-installed but that didn't work for me.

First thing: The "ESPHOME: <NSPanel-Name>_upload_tft" don't contains a url field, in ESPhome i have a seperate one for that: "ESPHOME: <NSPanel-Name>_upload_tft_url". But in either way the panel doesnt do anything except for showing the "Waiting for content..." screen from Lovelace UI. Logs showing the "Reading from UART timed out at byte 0"-error.

I'm also curious about the right order. To get the Upload TFT options, at least i most flash the old Tasmota with ESPhome, but that sounds like a wrong order because the warning says pretty clear: IMPORTANT! If you have already installed the NSPanel with my version, you must first fully install "Lovelace ui" again."

I am a bit confused about the right order and steps to take, to get my NSpanel from Tasmota to ESPHome again.

Now i followed the steps from https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-First-Steps---Installation-and-Setup and get stucked in the upload blank tft step.

Could you please help me?

Kind regards,

Smokey

**Comment by deejaybeam** [comment]
Hi
Yes there is a typo: _upload_tft should be _upload_tft_url

2nd should be better described, if you have lovelace UI installed, than you need to install the blank.tft with tasmota at first just before installing ESPHome and update/install via ESPHome-Services

Than do all the other steps to install this solution.

**Comment by Black-Smokey** [comment]
Hi,

Thank you for your reply. I've founded the same solution in the meanwhile.

I followed this steps:

- Be sure there is a Tasmota install on the device
- Login into the Tasmota console
- Go to "Consoles"
- Open the console (not the Berry Console)
- Use this command: FlashNextion http://<LOCAL_IP_HA>:8123/local/nspanel_blank.tft (When file is stored in www)

Run the command and after finishing the screen will show a white screen. After that, follow the instruction for ESPHome. (i also tried to use the above steps to flash the nspanel_eu.tft, that works fine to, but i have only tested it after i flashed the blank file).
